From dbf7b227cd16b8b90abffc8618e670a3e710ee87 Mon Sep 17 00:00:00 2001
From: Walter Dal Mut <walter.dalmut@corley.it>
Date: Wed, 26 Aug 2026 11:25:10 +0200
Subject: [PATCH 12/57] ARM: configs: rk3506_minimal: Correct CLK_RK3506
 dependency comment

The symbol depends on "ARM || COMPILE_TEST", not on ARM alone. Comment
only, no change to the merged configuration.

Signed-off-by: Walter Dal Mut <walter.dalmut@corley.it>

(cherry picked from commit b39872f266c3d90db1376bd99cd5311d6e3aa158)

diff --git a/arch/arm/configs/rk3506_minimal.config b/arch/arm/configs/rk3506_minimal.config
index 419854898..be8903a50 100644
--- a/arch/arm/configs/rk3506_minimal.config
+++ b/arch/arm/configs/rk3506_minimal.config
@@ -17,8 +17,9 @@
 CONFIG_ARCH_MULTI_V7=y
 CONFIG_ARCH_ROCKCHIP=y
 
-# Clock and reset controller. CLK_RK3506 is "default y" with "depends on ARM"
-# (drivers/clk/rockchip/Kconfig:103), listed here to pin the intent.
+# Clock and reset controller. CLK_RK3506 is "default y" and depends on
+# "ARM || COMPILE_TEST" (drivers/clk/rockchip/Kconfig:103), so it would be
+# picked up anyway; listed here to pin the intent.
 CONFIG_COMMON_CLK_ROCKCHIP=y
 CONFIG_CLK_RK3506=y
 
